src/fu-util.c

```diff
@@ -2657,6 +2682,42 @@ fu_util_get_blocked_firmware (FuUtilPrivate *priv, gchar **values, GError **erro
 	return TRUE;
 }
 
+static void
+fu_util_show_plugin_warnings (FuUtilPrivate *priv)
+{
+	FwupdPluginFlags flags = FWUPD_PLUGIN_FLAG_NONE;
+	g_autoptr(GPtrArray) plugins = NULL;
+
+	/* get plugins from daemon, ignoring if the daemon is too old */
+	plugins = fwupd_client_get_plugins (priv->client, NULL, NULL);
+	if (plugins == NULL)
+		return;
+
+	/* get a superset so we do not show the same message more than once */
+	for (guint i = 0; i < plugins->len; i++) {
+		FwupdPlugin *plugin = g_ptr_array_index (plugins, i);
+		if (!fwupd_plugin_has_flag (plugin, FWUPD_PLUGIN_FLAG_USER_WARNING))
+			continue;
+		flags |= fwupd_plugin_get_flags (plugin);
+	}
+
+	/* never show these, they're way too generic */
+	flags &= ~FWUPD_PLUGIN_FLAG_DISABLED;
+	flags &= ~FWUPD_PLUGIN_FLAG_NO_HARDWARE;
+
+	/* print */
+	for (guint i = 0; i < 64; i++) {
+		const gchar *tmp;
+		if ((flags & ((guint64) 1 << i)) == 0)
+			continue;
+		tmp = fu_util_plugin_flag_to_string ((guint64) 1 << i);
+		if (tmp == NULL)
+			continue;
+		/* TRANSLATORS: this is a prefix on the console */
+		g_print ("%s: %s\n", _("WARNING"), tmp);
```

Comment:
I think this should be noiser - in red on the console.

Reply:
Done!